lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -9,23 +10,76 @@ CFCRandomSpawn.spawnPointRankings = CFCRandomSpawn.spawnPointRankings or {}
 
 local function getMeasurablePlayers()
     local measurablePlayers = {}
-    for _, ply in pairs( player.GetHumans() ) do
-        if ( ply:Alive() ) then
+    for _, ply in pairs( player.GetAll() ) do
```

Comment:
This *might* cause issues if any kind of system is ever gonna use bots, i get that this was used for testing, not sure if this will ever be an issue though.